### PR TITLE
okta: add grantedScopes, grantType, clientSecret and requestedScopes fields from debug data

### PR DESCRIPTION
**Pull request by efd6**
## Proposed commit message

See title.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## Author's Checklist

- [ ] @terrancedejesus Please provide text for the fields' definition docs.

## Related issues

- For #11049

### CODE REVIEW

File: packages/okta/data_stream/system/elasticsearch/ingest_pipeline/no_use_flattened_debug.yml [hunk 1]
```diff
@@ -137,6 +137,26 @@ processors:
       target_field: okta.debug_context.debug_data.threat_suspected
       ignore_missing: true
       ignore_failure: true
+  - rename:
+      field: json.debugContext.debugData.clientSecret
+      target_field: okta.debug_context.debug_data.client_secret
+      ignore_missing: true
+      ignore_failure: true
```
Comment (kcreddy) [inline packages/okta/data_stream/system/elasticsearch/ingest_pipeline/no_use_flattened_debug.yml:144]: Could remove `ignore_failure: true`.

**Review by kcreddy — COMMENTED** [review]
The issue also talks about making `okta.target` queryable as it contains affected user or app. I don't see any references for it in this PR. Unlike `debugContext`,[`target`](https://developer.okta.com/docs/reference/api/system-log/#target-object) seems to have defined structure.

File: packages/okta/data_stream/system/elasticsearch/ingest_pipeline/no_use_flattened_debug.yml (hunk 1 above)
Comment (efd6) [reply packages/okta/data_stream/system/elasticsearch/ingest_pipeline/no_use_flattened_debug.yml:144]: I'd prefer to leave this.

**Comment by efd6** [comment]
The handling of the `okta.target` will be in another PR.

**Review by kcreddy — APPROVED** [review]

**Comment by terrancedejesus** [comment]
@efd6 - Thank you for taking the time to get these changes in! I see that these changes are for `2.13.0` so I will review them accordingly.

**Pull request merged**